.github/workflows/update_metadata.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow updates the check_metadata.sh script, which compares the current
+# size of build artifacts against their size in the previous version of Blockly.
+
+name: Update Metadata
+
+on: [workflow_dispatch]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [16.x]        
```

Comment:
As you're just running on a single OS / `node` version combo there's no need for a `matrix` and thus no need for for a `strategy` either.

Reply:
Done.